### ISSUE TITLE
Some edits

### ISSUE THREAD

**Issue by candroid-man** [pluja/awesome-privacy#274]
Add:

[Magic Earth](https://www.magicearth.com/)
[Neo Launcher](https://github.com/NeoApplications/Neo-Launcher)
[Send](https://github.com/timvisee/send)
[ProtonVPN](https://github.com/ProtonVPN)

Remove:

ClearURLs, Decentraleyes, Cookie Autodelete, and Smart Refer (all of these are now included in mainstream browsers)

**Comment by pluja** [comment]
Hi, thank you. Send by Tim Visée is already on the list.

I will need to review the other suggestions!

**Comment by supleed2** [comment]
I just looked into this as I still use ClearURLs, and it seems uBlock can do something similar, where filter lists can remove query parameters.
One example in the provided filter lists is: `Privacy > AdGuard URL Tracking Protection`
Is there an appropriate section for such suggestions, as I think only Firefox has made ClearURLs unnecessary and it's still useful on Chromium browsers.

**Comment by TalOrenshtein** [comment]
@pluja any reason ProtonVPN isn't on the list?

**Comment by pluja** [comment]
@TalOrenshtein just added ProtonVPN

**Comment by monokrome** [comment]
> ClearURLs, Decentraleyes, Cookie Autodelete, and Smart Refer (all of these are now included in mainstream browsers)

I don't think that these are in all mainstream browsers and also don't think that is significant enough reason to remove them. Browsers can remove these features as easily as they added them.

**Comment by fuzunspm** [comment]
magic earth should be added
